Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -82,6 +82,9 @@ import BraintreeCore
     /// Optional: If set to `true`, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement during checkout. Defaults to `false`.
     public var requestBillingAgreement: Bool
 
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
```

Comment:
Linking this comment from the[ Android PR](https://github.com/braintree/braintree_android/pull/806#discussion_r1371907212) to align docstrings across platforms. I think the one written here is a bit more descriptive!